estate/views/estate_property_offer_views.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="estate_property_offer_action" model="ir.actions.act_window">
+        <field name="name">Property Offers</field>
+        <field name="res_model">estate.property.offer</field>
+        <field name="view_mode">list,form</field>
+        <field name="domain">[('property_type_id', '=', active_id)]</field>
+    </record>
+
+
+    <record id="estate_property_offer_view_list" model="ir.ui.view">
+        <field name="name">estate.property.offer.list</field>
+        <field name="model">estate.property.offer</field>
+        <field name="arch" type="xml">
+            <list editable="bottom" decoration-success="status == 'accepted'" decoration-danger="status == 'refused'">
+                <field name="price"/>
+                <field name="partner_id" width="200"/>
+                <field name="validity" string="Validity (days)" width="100%"/>
+                <field name="date_deadline"/>
+                <button name="action_accept_offer" type="object" title="Accept" icon="fa-check" invisible="status in ['accepted', 'refused'] or property_state in ['offer_accepted', 'sold', 'cancelled']"/>
+                <button name="action_refuse_offer" type="object" title="Refuse" icon="fa-times" invisible="status in ['accepted', 'refused'] or property_state in ['offer_accepted', 'sold', 'cancelled']"/>
```

Comment:
The invisible condition is really long, in that case I'd create a computed field for the condition instead